node/src/reactor/main_reactor/keep_up.rs

```diff
@@ -321,8 +318,8 @@ impl MainReactor {
         self.attempts += 1;
         warn!(
             %error,
-            "Historical: failed leap, remaining attempts: {}",
-            self.max_attempts.saturating_sub(self.attempts)
+            remaining_attempts = self.max_attempts.saturating_sub(self.attempts),
+            "Historical: failed leap",
```

Comment:
Should we make this lowercase `historical` for consistency with the log above?

Also, same below.

Reply:
done https://github.com/casper-network/casper-node/pull/3507/commits/b67de88156f797f27ab60dcadf6c39944d845711